### ISSUE TITLE
Is it possible to set the `tooltipContainerBounds` after the root has been created?

### ISSUE THREAD

**Issue by lindapaiste** [amcharts/amcharts5#1156]
Can you confirm whether or not it's possible to change the `tooltipContainerBounds` on an active `Root`?  Looking at the source code it seems like it would be either impossible or very difficult to change this properly, as the `tooltipContainerBounds` setting is used internally in the `Root`.  For example:

https://github.com/amcharts/amcharts5/blob/3085229255d343c2dc1c80e10b84e7ed697aad99/src/.internal/core/Root.ts#L607-L609

I can do something like this:
```
root.tooltipContainer.set('layerMargin', { top: 0, left: 0, right: 0, bottom: 200 });
```
But then the `Root` instance wouldn't be aware of those boundaries, right?

**Comment by martynasma** [comment]
No, I'm afraid. `Root` settings are one-time. You can set them when creating the `Root` element only.
